src/components/FilterGroup.tsx

```diff
@@ -0,0 +1,111 @@
+import { useMemo } from 'react';
+import { CompositionMethod } from '../hooks';
+import {
+  CheckboxOption,
+  CheckboxOptionProps,
+  CollapsibleLabel,
+  CollapsibleSection,
+  FilterGroup as FilterGroupCompoundComponent,
+  SearchInput
+} from './Filters';
+
+/**
+ * The CSS class interface for FilterGroup.
+ *
+ * @public
+ */
+export interface FilterGroupCssClasses {
+  searchInput?: string,
+  optionsContainer?: string,
+  option?: string,
+  optionInput?: string,
+  optionLabel?: string
+}
+
+/**
+ * The configuration data for a filter option.
+ *
+ * @public
+ */
+export type FilterOptionConfig = Omit<CheckboxOptionProps, 'customCssClasses' | 'cssCompositionMethod'>;
+
+/**
+ * Props for the FilterGroup component.
+ *
+ * @public
+ */
+export interface FilterGroupProps {
+  /** The fieldId corresponding to the filter group. */
+  fieldId: string,
+  /** {@inheritDoc FilterOptionConfig} */
+  filterOptions: FilterOptionConfig[],
+  /** The displayed label for the filter group. */
+  title: string,
+  /** Whether or not the filter is collapsible. Defaults to true. */
+  collapsible?: boolean,
+  /**
+   * If the filter group is collapsible, whether or not it should start out
+   * expanded. Defaults to true.
+   */
+  defaultExpanded?: boolean,
+  /** Whether or not to display a text input to search for filter options. */
+  searchable?: boolean,
+  /** CSS classes for customizing the component styling. */
+  customCssClasses?: FilterGroupCssClasses,
+  /** {@inheritDoc CompositionMethod} */
+  cssCompositionMethod?: CompositionMethod
+}
+
+/**
+ * Renders a group of selectable filters with support for searching and collapsing.
+ */
+export function FilterGroup({
```

Comment:
I don't think we should have two components with the same name

Reply:
This component basically provides the common layout we use for facets and filters so maybe something along those lines...? Honestly not sure

Reply:
I think it makes sense to have this as` FiltersGroup`, product seems to use similar term to describe this group of selectable filters. I will update the components in `Filters` to `FiltersGroupContextProvider` `FacetsContextProvider` and ``StaticFiltersContextProvider``